vme/src/act_info.cpp

```diff
@@ -23,9 +23,9 @@ const char *in_string(const unit_data *ch, const unit_data *u)
 {
     static std::string in_str;
     in_str.clear();
-    while ((u = UNIT_IN(u)))
+    while ((u = u->getMyContainer()))
```

Comment:
I would be very happy if we could rename getMyContainer() to unitIn(). It's so embedded in my mind that I*ll struggle a bit with getMyContainer. :-)

Reply:
Done